sched/task/task_exithook.c

```diff
@@ -80,7 +80,7 @@ static inline void nxtask_exitstatus(FAR struct task_group_s *group,
 }
 #else
 
-#  define nxtask_exitstatus(group,status)
+#  define nxtask_exitstatus(group,status) (group)->tg_exitcode = (status);
```

Comment:
Better to `# define nxtask_exitstatus(group,status) ((group)->tg_exitcode = (status))` to catch possible missing `;` after `nxtask_exitstatus(a,b)`

Reply:
Yes. I was going to do do { ... } while(0) but for some reason did this brainfart instead.

Need to come back to this later.

Reply:
Yes. `do {} while (0)` is even better